rllib/utils/exploration/per_worker_epsilon_greedy.py

```diff
@@ -54,10 +54,12 @@ def reset_schedule(self,
                        epsilon_schedule=None
                        ):
         epsilon_schedule = None
-        if self.num_workers > 0:
+        if self.num_workers == 1:
```

Comment:
Just curious -- what would happen if we don't add this condition:)

Reply:
We get a zero division error if we set num_workers to 1. So I had to handle that condition separately.